modules/solid_mechanics/src/materials/ReturnMappingModel.C

```diff
@@ -42,7 +44,9 @@ ReturnMappingModel::ReturnMappingModel(const InputParameters & parameters,
     _effective_inelastic_strain(
         declareProperty<Real>("effective_" + inelastic_strain_name + "_strain")),
     _effective_inelastic_strain_old(
-        declarePropertyOld<Real>("effective_" + inelastic_strain_name + "_strain"))
+        declarePropertyOld<Real>("effective_" + inelastic_strain_name + "_strain")),
+    _max_inelastic_incr(isParamValid("max_inelastic_incr") ? getParam<Real>("max_inelastic_incr")
+                                                           : 0.)
```

Comment:
I'm coming into this late since this has been merged, but I stumbled on this code, and see a few issues.  First off, this use of isParamValid doesn't really make sense. It looks like it's setting it to 0 if it's not set, but then that parameter has a default value of 1e-4, so it would always use that as the default.